<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Labyrinth size and attributes</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QSpinBox" name="height_box"/>
     </item>
     <item row="3" column="3">
      <widget class="QRadioButton" name="random_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="start_x_box"/>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="goal_y_label">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QSpinBox" name="goal_y_box"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="goal_x_label">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QPushButton" name="show_button">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="start_label">
       <property name="text">
        <string>Start :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QSpinBox" name="start_y_box"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="height_label">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="width_label">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLabel" name="random_label">
       <property name="text">
        <string>Choose start and goal position randomly?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="start_y_label">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="goal_x_box"/>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="random_yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="width_box"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="start_x_label">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="goal_label">
       <property name="text">
        <string>Goal:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
